com.unity.formats.fbx/Documentation~/export-record-in-play-mode-recorder-clip.md

```diff
@@ -0,0 +1,32 @@
+# Record FBX animations via a Timeline Recorder Track
+
+Set up one or multiple Recorder clips in a Timeline Recorder Track to activate the recording of GameObject's animations to FBX at specific time or frame intervals in Play mode.
+
+>**Note:** This scenario requires to have both the [Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@latest) and [Timeline](https://docs.unity3d.com/Packages/com.unity.timeline@latest) packages installed in addition to the FBX Exporter package.
+
+To record an FBX animation through a Timeline Recorder Track:
+
+1. Open the Timeline where you want to set up the Recorder Track.
+
+2. In the Timeline window, click on the Add (**+**) button and select **UnityEditor.Recorder.Timeline** > **Recorder Track**.
+
+3. Right-click in the **Recorder Track** you just added and select **Add Recorder Clip**.
+
+4. Select the added **Recorder Clip**, and in the Inspector, in the **Selected recorder** dropdown list, select **FBX**.
+
+6. In the **Capture** section, set the **GameObject** to export the animation from.
```

Comment:
I don't see a Capture section in the image for the recorder settings, should it be input?